about.html

```diff
@@ -0,0 +1,89 @@
+    <div>
+      <h1 class="title">
+        digital master plan global summit 2023
+      </h1>
+  
+      <p class="para">
+        The global summit brings together the community of experts, academics, and activist who comprise the Creative Commons affliate network in a different country every two, This year, we hope to expand our invitation list including organizations and individuals.
+      </p>
+  
+      <div>
+        <p class="txt-details">
+          please contact us per any Email for any Further questions
+          about Digital Master Plan<br>
+          <span class="address-info">digital@info.ac.ke</span>
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="section container d-flex-column align-items-center justify-content-center">
+    <h3 class="logo-title header-3 text-align-center">
+      digital master plan global summit 2023 Logo
+    </h3>
+    <p class="logo-text-info txt-details text-align-center">
+      The logo of Digital Master Plan 2023 was decided through the logo competition from 8, June to 7, July. After online public voting and Selection Committee's judging the process, the logo from Naresh Agrawal won the prize. After consultation with Mr. Agrawal we optimized the winning logo for this year's summit as following
+    </p>
+
+    <div class="logo-icon d-flex justify-content-center align-items-center">
+      <img class="dmp-logo" src="icons/Digital innovation_0.png" alt="logo">
+
+      <p class="logo-info">
+        Digital Master<br> Plan 2023
+      </p>
+    </div>
+  </section>
+  <section class="section container d-flex-column d-ds-flex-column justify-content-center align-items-center">
+    <h3 class="header-3 text-align-center">
+      see the past digital master plan Summits
+    </h3>
+    <p class="past-events-info">
+      Take a look at the last two Digital master plan which took place at Mombasa and Kisumu.
+    </p>
+    <div class="past-events d-flex justify-content-center">
+      <div class="event-container">
+        <img class="icon-fluid past-event-photo" src="https://capria.vc/wp-content/uploads/2022/04/kenya-economic-update-accelerating-kenyas-digital-economy-780x439-1.jpeg" alt="">
+        <div class="centered">
+          <p class="centered-text text-align-center">
+            2019
+          </p>
+          <p>
+            Digital Master Plan in Mombasa
+          </p>
+        </div>
+      </div>
+      <div class="event-container">
+        <img class="icon-fluid past-event-photo" src="https://media.istockphoto.com/id/1402667894/photo/energy-lecture-screen.jpg?b=1&s=170667a&w=0&k=20&c=4IN-1IUvtrMK3h8fnrWf490o5KzoXfDFd6gL3Rbn5Ak=" alt="">
+        <div class="centered-text centered">
+          <p class="text-align-center">
+            2021
+          </p>
+          <p>
+            Digital Master Plan in Kisumu
+          </p>
+        </div>
+      </div>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] Please consider fixing the images in this section and add images as given in the design guidelines. 
- [ ] I noticed that you missed the line under the section's heading. Please make sure to add it. 
- [ ] The text on the images is difficult to read. Please make sure that you add it according to the design guidelines so that it is easier to read. I know you can do that 💪
- [ ] Please make sure that the text is aligned to the center as given in the design guidelines 🎯


### _Desktop version 🔽_
Your design | Design guidelines |
--- | --- | 
![image](https://user-images.githubusercontent.com/106119105/217345990-b9b8ff56-88b6-4539-a428-e6bb3bf50c10.png) |![image](https://user-images.githubusercontent.com/106119105/217345847-92f7be23-5193-4f5b-88db-41cd94aa783a.png) | 

## **_Mobile version 🔽_**
Your design | Design guidelines |
--- | --- | 
![image](https://user-images.githubusercontent.com/106119105/217346945-fe4565e6-1660-4680-88f4-3fcb4b0c3f8d.png) | ![image](https://user-images.githubusercontent.com/106119105/217347008-c088c448-37a0-4bc5-9432-c42c7dde6bb6.png) |